### PR TITLE
Adjust API to use plural nouns

### PR DESCRIPTION
**Pull request by vtereso**
# Changes
This PR is to address the current API to align with RESTful best practice to use plural nouns.

Adjust back-end and front-end router to use corrected plural syntax and corresponding tests. 
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by vtereso** [comment]
@AlanGreene Does this look good from your side?

File: src/api/index.test.js [hunk 1]
```diff
@@ -56,22 +56,22 @@ describe('getAPIRoot', () => {
 
 describe('getAPI', () => {
   it('returns a URI containing the given type', () => {
-    const uri = getAPI('pipeline');
-    expect(uri).toContain('pipeline');
+    const uri = getAPI('pipelines');
```
Comment (AlanGreene) [inline src/api/index.test.js]: I don't see any update to the implementation in src/api/index.js. The unit tests should have caused the build to fail.

**Comment by vtereso** [comment]
@AlanGreene Thanks. I made changes and did an add all, but they didn't seem to be added.

**Comment by vdemeester** [comment]
@vtereso you're gonna need to rebase against master to fix the ci :bowing_man:

**Comment by vtereso** [comment]
This PR should be merged alongside https://github.com/tektoncd/experimental/pull/27

**Comment by mnuttall** [comment]
/lgtm
Previous comment isn't being picked up

**Pull request merged**